scrape_s3_redirects.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import boto3
+import datetime
+import time
+import json
+
+
+def extractKey(s3_object)-> list:
+    return s3_object.key
+
+def writeObjectKeys(bucket)-> list:
+    bucketObj = boto3.session.Session().resource("s3").Bucket(bucket)
+    s3_objects = list(bucketObj.objects.all())
```

Comment:
do we need `all` objects? seems like we only need html files for redirects CMIIW

Reply:
I wasn't sure so I included all objects to be safe..

Reply:
It would definitely cut down on time to only look through the html files,  but I don't want to miss anything, do you have any suggestions on how to best verify your assumption?